pages/index.js

```diff
@@ -1,38 +1,53 @@
 import Head from "next/head";
 import { useState } from "react";
 import Map from "@/components/Map/Map";
-import SearchPlaces from "@/components/SearchPlaces/SearchPlaces";
+import SearchBar from "@/components/SearchBar/SearchBar";
 
 // Change Token before git push!!!
 const accessToken =
-  "pk.eyJ1Ijoia2V0bmVrIiwiYSI6ImNsaDVybDl5azA0ZzYzZ21wcmJvZnlxeGYifQ.7xOZhYeVEcLNK833qEUjUw";
+  "pk.eyJ1Ijoia2V0bmVrIiwiYSI6ImNsaDdxaDgzNzAxN2wzZnFtaXQ0NW41NmMifQ.uKSRkOjJ2_Enua9-5gVOIQ";
 
 export default function Home() {
+  const [zoom, setZoom] = useState(9);
   const [lng, setLng] = useState(10.000654);
   const [lat, setLat] = useState(53.550341);
-  const [zoom, setZoom] = useState(9);
   const [center, setCenter] = useState(null);
+  const [searchValue, setSearchValue] = useState("");
+  const [searchResults, setSearchResults] = useState([]);
+  const [inputPlaceholder, setInputPlaceholder] = useState("Search for places");
 
+  // fetch data
   const getNewDestination = async (url) => {
     try {
       const response = await fetch(url);
       if (!response.ok) {
         console.error("Loading new destination went wrong!");
       }
       const data = await response.json();
-      setCenter(data?.features[0]?.center);
+      setSearchResults(data.features);
     } catch (error) {
       console.error(error);
     }
   };
 
+  // all handler
   const handleSearchSubmit = (event) => {
     event.preventDefault();
-    const form = event.target;
+    setCenter(searchResults[0]?.center);
```

Comment:
This leads to unexpected behavior: I typed in 'Münster' and the results-array shows at first position: "Münster Flughafen". Pressing enter leads to 'Flughafen-Flughafen' - not to 'Münster-City'.

Reply:
Thank's Roland, I try my best.😊 I will make a deep dive into the docs again. Maybe I can find something new to fix it.

Reply:
Morning Roland, 
I found a way to fix that, but when I set this new parameter I'm not be able to search for special places like airports, restaurants etc. anymore. The reason why the airport comes first is the rating of the api. It seems like the Münster Airport is more popular as the city Münster in mapbox eyes. So I decided to life with that because I don't want to lose the possibility to search for special places.

![Screenshot 2023-05-04 at 08 42 30](https://user-images.githubusercontent.com/113082466/236129282-5537bb29-fd4d-4361-9561-5cd2197b92ff.png)